src/components/ContextualMenu/ContextualMenuDropdown/ContextualMenuDropdown.tsx

```diff
@@ -221,48 +200,42 @@ const ContextualMenuDropdown = <L,>({
 
   return (
     <span
-      className={wrapperClass}
-      style={positionStyle as React.CSSProperties}
+      className={classNames("p-contextual-menu__dropdown", dropdownClassName)}
+      id={id}
+      aria-hidden={isOpen ? "false" : "true"}
+      aria-label={Label.Dropdown}
+      ref={dropdown}
+      style={{
+        ...(positionStyle as React.CSSProperties),
```

Comment:
I think there is no need to cast here.
```suggestion
        ...positionStyle,
```